### PR TITLE
Add structural variant case_list

### PR DESCRIPTION
**Pull request by rmadupuri**

### CODE REVIEW

File: generate_case_list/generate_case_lists.py [hunk 1]
```diff
@@ -194,7 +194,7 @@ def get_case_list_from_staging_file(study_dir, staging_filename, verbose):
                     break # got case ids from header, don't read the rest of the file
                 else:
                     # we know at this point one of these columns exists, so no fear of ValueError from index method
-                    id_column_index = values.index(MUTATION_CASE_ID_COLUMN_HEADER) if MUTATION_CASE_ID_COLUMN_HEADER in values else values.index(SAMPLE_ID_COLUMN_HEADER)
+                    id_column_index = values.index(MUTATION_CASE_ID_COLUMN_HEADER) if MUTATION_CASE_ID_COLUMN_HEADER in values else [x.upper() for x in values].index(SAMPLE_ID_COLUMN_HEADER)
```
Comment (yichaoS) [inline generate_case_list/generate_case_lists.py]: So this allows "sample_id" & "sample_ID" right? How about "sampleid"?

File: generate_case_list/case_list_conf.txt [hunk 2]
```diff
@@ -4,17 +4,17 @@ cases_cna.txt	data_CNA.txt	<CANCER_STUDY>_cna	all_cases_with_cna_data	<CANCER_ST
...
-cases_complete.txt	data_RNA_Seq_expression_median.txt&data_CNA.txt&data_mutations_extended.txt	<CANCER_STUDY>_3way_complete	all_cases_with_mutation_and_cna_and_mrna_data	<CANCER_STUDY>	Complete samples	Samples with mutation, CNA and expression data (<NUM_CASES> samples)
-cases_complete.txt	data_mRNA_median_Zscores.txt&data_CNA.txt&data_mutations_extended.txt	<CANCER_STUDY>_3way_complete	all_cases_with_mutation_and_cna_and_mrna_data	<CANCER_STUDY>	Complete samples	Samples with mutation, CNA and expression data (<NUM_CASES> samples)
-cases_complete.txt	data_expression_Zscores.txt&data_CNA.txt&data_mutations_extended.txt	<CANCER_STUDY>_3way_complete	all_cases_with_mutation_and_cna_and_mrna_data	<CANCER_STUDY>	Complete samples	Samples with mutation, CNA and expression data (<NUM_CASES> samples)
+cases_complete.txt	data_RNA_Seq_v2_mRNA_median_Zscores.txt&data_CNA.txt&data_mutations_extended.txt	<CANCER_STUDY>_3way_complete	all_cases_with_mutation_and_cna_and_mrna_data	<CANCER_STUDY>	Complete samples	 Samples with mutation, CNA and expression data (<NUM_CASES> samples)
```
Comment (yichaoS) [inline generate_case_list/case_list_conf.txt]: what's changed here?

File: generate_case_list/case_list_conf.txt [hunk 3]
```diff
@@ -4,17 +4,17 @@ cases_cna.txt	data_CNA.txt	<CANCER_STUDY>_cna	all_cases_with_cna_data	<CANCER_ST
...
-cases_methylation_promoters_wgbs.txt	data_methylation_promoters_wgbs.txt	<CANCER_STUDY>_methylation_promoters_wgbs	all_cases_with_methylation_data	<CANCER_STUDY>	Tumor samples with methylation data (Promoters, WGBS)	All samples with methylation (Promoters, WGBS) data (<NUM_CASES> samples)
-cases_methylation_genebodies_wgbs.txt	data_methylation_genebodies_wgbs.txt	<CANCER_STUDY>_methylation_genebodies_wgbs	all_cases_with_methylation_data	<CANCER_STUDY>	Tumor samples with methylation data (Gene Bodies, WGBS)	All samples with methylation (Gene Bodies, WGBS) data (<NUM_CASES> samples)
+cases_methylation_promoters_hmEPIC.txt	data_methylation_promoters_hmEPIC.txt	<CANCER_STUDY>_methylation_promoters_hmEPIC	all_cases_with_methylation_data	<CANCER_STUDY>	 Tumor samples with methylation data (Promoters, hmEPIC) 	 All samples with methylation (Promoters, hmEPIC) data (<NUM_CASES> samples)
```
Comment (yichaoS) [inline generate_case_list/case_list_conf.txt]: And here also?

File: generate_case_list/case_list_conf.txt (hunk 2 above)
Comment (rmadupuri) [reply generate_case_list/case_list_conf.txt]: I think extra spaces have been removed.

File: generate_case_list/generate_case_lists.py (hunk 1 above)
Comment (rmadupuri) [reply generate_case_list/generate_case_lists.py]: This is according to the SV file format in `msk_impact_2017` study.

File: generate_case_list/case_list_conf.txt (hunk 3 above)
Comment (rmadupuri) [reply generate_case_list/case_list_conf.txt]: Same here. There were extra spaces that got removed. Nothing changed with the data.

**Pull request merged**